app/session.go

```diff
@@ -71,6 +71,17 @@ func (a *App) GetSession(token string) (*model.Session, *model.AppError) {
 		return nil, model.NewAppError("GetSession", "api.context.invalid_token.error", map[string]interface{}{"Token": token}, "", http.StatusUnauthorized)
 	}
 
+	if utils.IsLicensed() && *utils.License().Features.Compliance &&
```

Comment:
Probably a marginal effect but can we check `SessionIdleTimeoutInMinutes` > 0 first? It's the 99% case that will be false and we don't have to check anything else.

Reply:
Done